resources/constructions.rb

```diff
@@ -1004,36 +1004,39 @@ def self.apply_skylight(runner, model, subsurfaces, constr_name, weather,
   end
 
   def self.apply_partition_walls(runner, model, surfaces, constr_name,
-                                 drywall_thick_in, frac_of_ffa)
+                                 drywall_thick_in, frac_of_ffa, basement_frac_of_cfa, cond_base_surfaces)
 
-    spaces = Geometry.get_conditioned_spaces(model.getSpaces)
+    living_space = Geometry.get_conditioned_space(model.getSpaces)
```

Comment:
Came across this part which was more intended for separate zone approach. Did some cleanup, wondering maybe we could pass the conditioned zone object from `measure.rb` instead of doing extra search every time?